nimbleparse/src/main.rs

```diff
@@ -108,13 +108,13 @@ fn main() {
         .optopt(
             "r",
             "recoverer",
-            "Recoverer to be used (default: cpctplus)",
+            "Recoverer to be used (default: cpctplus unless specified in grammar)",
```

Comment:
This text could perhaps be more leading towards the user finding how to specify it.
E.g. "unless specified in the %grmtools section of the grammar", but I feared making it too long
and I don't see any obvious way it gets printed, so I stuck with "unless specified in the grammar"
which is already way longer than the current text.